server/controllers/profile.js

```diff
@@ -40,35 +40,11 @@ exports.post = asyncHandler(async (req, res) => {
 });
 
 exports.patch = asyncHandler(async (req, res) => {
-  const {
-    firstName,
-    lastName,
-    gender,
-    birthday,
-    phone,
-    address,
-    description,
-    availability,
-    available,
-  } = req.body;
   const id = req.user.id;
   const idExists = await Profile.findOne({ user: id });
 
   if (idExists) {
-    const update = await Profile.updateOne(
-      { user: id },
-      {
-        firstName,
-        lastName,
-        gender,
-        birthday,
-        phone,
-        address,
-        description,
-        availability,
-        available,
-      }
-    );
+    const update = await Profile.updateOne({ user: id }, req.body);
```

Comment:
This must be an accidental change - if I'm not mistaken, the new changes are from an older version of this file 🤔

Reply:
Not an accidental change actually! We had a bug due to the way the patch request was implemented. we had values being set to null if they were not specifically set this way works! 
Do you see any issues that would arise from doing it this way?

Reply:
I'm going to push these changes, I'll hotfix once I have more information!